changelog

```diff
@@ -1,3 +1,59 @@
+Version 0.9.10:
+  * Bug fixes:
+    * Typo fixes. (@Darcman99, @Kryes-Omega, @MCOfficer, @petervdmeer, @realityforge, @SolraBizna, @tehhowch, @warp-core, @Zitchas)
+    * Terraforming 6 will no longer offer on planets that make it impossible to complete before the deadline. (@Comnom)
+    * Pug and Free Worlds ships will now properly be hostile toward each other during FWC Pug 3B. (@Amazinite)
+    * The Timer Ship will now properly destroy itself. (@tehhowch)
+    * Corrected the descriptions of the Sidewinder and Meteor missiles to refer to the proper capacity of their launchers. (@Amazinite)
+    * Completing the Plasma Cannon retrieval mission will now properly continue the Remnant storyline. (@Zitchas)
+    * Terraforming 6 will no longer offer on uninhabited planets. (@Amazinite)
+    * Mystery Cube 2 escorts now have the proper governments. (@Amazinite)
+    * The outfitter will no longer be able to load more ammo than can be held. (@jostephd)
+    * Outfits that don't depreciate will no longer show a depreciated cost in the boarding panel. (@tehhowch)
+    * Ships with zero cargo space will no longer be overloaded with cargo. (@tehhowch)
+    * Various boarding missions will no longer offer on ships without any crew. (@tehhowch)
+    * The Unwanted Cargo mission now correctly identifies when you have the translator. (@Amazinite)
+    * Paradise Fortune 1's on visit dialog no longer refers to Diana by name. (@petervdmeer)
+  * Game content:
+    * Black holes now have a landing message. (@Comnom)
+    * Pirate occupation spaceport missions will no longer cause the player to launch if the occupation is in another system. (@ZBok)
+    * Some Deep missions will no longer offer while the player is doing the main storyline. (@Amazinite)
+    * Reduced the speed of the Centipede. (@Amazinite)
+    * Remnant Tech Retrieval missions will no longer be blocked by other missions. (@Zitchas)
+    * EMP torpedoes now require a Remnant license to purchase. (@Zitchas)
+    * Nearly all missions that could possibly trigger an on visit dialog no have one, making it easier to tell why you aren't completing a mission when at the destination. (@Amazinite)
```

Comment:
I'm guessing this is supposed to be, "Nearly all missions that could possibly trigger an on visit dialog **now** have one"

Reply:
Thanks. Fixed.